technicalreport/introduction.tex

```diff
@@ -15,6 +15,12 @@ \subsubsection{ÖV-Güteklasse}
 \acs{ÖV}-Güteklassen geben Auskunft darüber, wie gut ein Standort mit dem öffentlichen Verkehr erschlossen ist.
 Dies ist wichtig, wenn es darum geht, den öffentlichen Verkehr zu optimieren, Siedlungsverdichtung nach Innen an geeigneten Lagen voranzutreiben oder Standortentscheide für publikumsintensive Anlagen so zu treffen, dass sie möglichst wenig zusätzlichen Autoverkehr verursachen.~\cite{oev-guteklasse-gr-defintion}
 
+\subsubsection{Haltestelle}
+\label{begriff:Haltestelle}
+
+Für den Zweck dieser Arbeit werden nur Bahnhöfe, Tram- und Bushaltestellen in die Berechnung einbezogen.
```

Comment:
In HAVAS sind auch noch Seilbahnen etc. drin. Glaube nicht, dass wir diese grundsätzlich ausschliessen können, da ländlichere Gebiete so ganz in der Berechnung benachteiligt werden.

Reply:
Sofern ich das sehe wird das in GR auch so gemacht. Es werden noch "manuell" Seilbahnen hinzugefügt, die eine Erschliessungsfunktion haben. Weiss nicht, ob wir dies automatisch prüfen können. (Diese werden dort wie Bushaltestellen behandelt)

Reply:
Besprechen wir das kurz morgen